packages/memory-inspector/src/browser/style/index.css

```diff
@@ -0,0 +1,753 @@
+    grid-auto-flow: column;
+    border-bottom: 1px solid hsla(0, 0%, 50%, 0.5);
+    border-top: 1px solid hsla(0, 0%, 50%, 0.5);
+    padding-bottom: var(--theia-ui-padding);
+}
+
+.view-group-go-button {
+    grid-row: 2 / 3;
+    height: 100%;
+}
+
+.t-mv-group {
+    display: grid;
+    grid-column-gap: var(--theia-ui-padding);
+    grid-row-gap: calc(var(--theia-ui-padding)/2);
+    align-items: center;
+}
+
+.t-mv-group .theia-select,
+.t-mv-group .theia-input {
+    width: 100%;
+    box-sizing: border-box;
+}
+
+.t-mv-group .multi-select-bar {
+    min-height: 19px;
+}
+
+.t-mv-group .multi-select-label {
+    line-height: 17px;
+}
+
+.t-mv-input-group.button-wrapper {
+    margin: 2px;
+    width: 100%;
+    display: block;
+    text-align: center;
+}
+
+.t-mv-input-group.button-wrapper .theia-button.main {
+    margin: var(--theia-ui-padding) 0 0 0;
+}
+
+table.t-mv-view {
+    align-self: center;
+    border-collapse: collapse;
+    border: none;
+}
+
+table.t-mv-view {
+    font-size: var(--theia-ui-font-size1);
+    font-weight: 400;
+    text-align: left;
+    padding: 10px;
+}
+
+table.t-mv-view td,
+table.t-mv-view th {
+    font-family: monospace;
+    padding-right: 15px;
+    white-space: pre;
+}
+
+table.t-mv-view .t-mv-view-row.t-mv-view-row-highlight {
+    border-bottom: 1px var(--theia-editorGroup-border) solid;
+}
+
+table.t-mv-view .t-mv-view-row:hover {
+    background-color: var(--theia-sideBar-background);
+}
+
+table.t-mv-view .t-mv-view-address {
+    font-weight: 700;
+    text-align: left;
+}
+
+.t-mv-settings-group-header.small-margin {
+    margin-bottom: var(--theia-ui-padding);
+    margin-top: var(--theia-ui-padding);
+}
+
+.t-mv-variable-label:not(:last-of-type)::after {
+    content: ',';
+    margin-right: var(--theia-ui-padding);
+    color: var(--theia-editor-foreground);
+}
+
+.mw-input-select {
+    position: relative;
+    height: 100%;
+}
+
+.mw-input-select .theia-input {
+    position: absolute;
+    z-index: 1;
+    width: calc(100% - 18px);
+}
+
+.mw-input-select .theia-input:focus {
+    outline: unset;
+}
+
+.mw-input-select .theia-select {
+    position: absolute;
+    z-index: 0;
+    height: calc(var(--theia-content-line-height) + 2px);
+}
+
+.mw-input-select .theia-select:focus {
+    outline: unset;
+}
+
+.memory-layout-widget {
+    display: flex;
+    flex-direction: column;
+}
+
+.memory-dock-panel {
+    flex-grow: 1;
+    margin: var(--theia-ui-padding);
+    margin-top: 0;
+}
+
+.memory-edit-button-container {
+    position: relative;
+    padding: var(--theia-ui-padding) 0;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    box-shadow: 0px 6px 6px 6px var(--theia-widget-shadow);
+}
+
+.memory-edit-button-container .memory-edit-error {
+    position: absolute;
+    width: 100%;
+    bottom: 0;
+    box-sizing: border-box;
+    white-space: normal;
+    padding: var(--theia-ui-padding);
+    background-color: var(--theia-editor-background);
```

Comment:
The fetch error has a different error color and is centered:
```
.t-mv-memory-fetch-error {
    margin: var(--theia-ui-padding);
    text-align: center;
    color: var(--theia-errorForeground);
}
```

Is it a conscious choice to have it different for write errors? I am fine with it, just want to make sure.

Reply:
I've aligned them with each other in favor of the styling on the write error. Prose text is better aligned left, I think.

Reply:
Sure, as long as it is consistent, I am fine with it, thank you!